app/models/app_problem_report_ticket.rb

```diff
@@ -44,4 +44,8 @@ def validates_email_if_can_reply
       errors.add(:email, "Please add an email address")
     end
   end
+
+  def spam?
```

Comment:
Oooh. Hmm. Let me double-check here. Are these forms expected to be only run inside the app, but this is here because it's technically possible to get to the endpoint (ie it's public, just not listed)? It's worth bearing in mind that there are some problems with giraffe fields where some browsers autofill them regardless, leading to things being false-positived as spam. So if the only expected legitimate use is in a browser you can control (ie the app), that's probably safe. But if not, you might want to double-check if this field is going to cause problems.

Reply:
Discussed briefly offline. I've since tested with the giraffe field and not found any issues, however will monitor and remove if any issues occur. A side note being, if there are issues then we probably should look into the use on the main [gov.uk contact form](https://www.gov.uk/contact/govuk).